controllers/drplacementcontrol_controller.go

```diff
@@ -808,17 +808,20 @@ func (r *DRPlacementControlReconciler) Reconcile(ctx context.Context, req ctrl.R
 	return r.reconcileDRPCInstance(d, logger)
 }
 
-func (r *DRPlacementControlReconciler) setProgressionAndUpdate(
-	ctx context.Context, drpc *rmn.DRPlacementControl, progressionStatus rmn.ProgressionStatus,
+func (r *DRPlacementControlReconciler) setDeletionStatusAndUpdate(
+	ctx context.Context, drpc *rmn.DRPlacementControl,
 ) error {
-	updated := updateDRPCProgression(drpc, progressionStatus, r.Log)
+	updated := updateDRPCProgression(drpc, rmn.ProgressionDeleting, r.Log)
```

Comment:
Update status.ObserverdGeneration as well to the meta generation here. [see](https://github.com/RamenDR/ramen/blob/39485b5adc51671842534660e287ce07503542fd/controllers/drplacementcontrol.go#L2094)

On a resource delete the resource generation is incremented, hence noting the generation in status is helpful as well.

Reply:
updated